spk/go2rtc/Makefile

```diff
@@ -1,6 +1,6 @@
 SPK_NAME = go2rtc
-SPK_VERS = 1.8.5
-SPK_REV = 4
+SPK_VERS = 1.9.2
+SPK_REV = 8
```

Comment:
it is not required to increment SPK_REV within a PR. So `SPK_REV = 5` is expected here.